proxy/src/proxy.rs

```diff
@@ -19,6 +23,10 @@ use crate::{
     util::{self, ControlFlow},
 };
 
+const LITERAL_ACCEPT_TEXT: &str = "proxy: Literal accepted by proxy";
+const LITERAL_REJECT_TEXT: &str = "proxy: Literal rejected by proxy";
+const COMMAND_REJECTED_TEXT: &str = "proxy: Command rejected by server";
```

Comment:
The actual text is up for discussion. But messages like these might help to identify IMAP messages that are not forwarded but instead generated by the proxy itself.